src/genie/libs/parser/nxos/show_vrf.py

```diff
@@ -177,7 +177,7 @@ def cli(self, vrf='', output=None):
             line = line.strip()
 
             # VRF-Name: VRF1, VRF-ID: 3, State: Up
-            p1 = re.compile(r'^VRF\-Name: +(?P<vrf>[\w\-]+), +'
+            p1 = re.compile(r'^VRF\-Name: +(?P<vrf>[\w\d\-\.\:]+), +'
```

Comment:
if it works, simply `\S+` might be better.
```suggestion
            p1 = re.compile(r'^VRF\-Name: +(?P<vrf>\S+), +'
```

Reply:
`\S+` accepts all ASCII characters, including, for instance, the `{`, `}`, `[`, `]` and `|`, which are not accepted by Cisco IOS (XR) as valid characters for a VRF. That's why I choose to expand on the regexes already present in the code.